packages/atlas-experiment-table/src/TableManager.js

```diff
@@ -58,7 +58,7 @@ export default class TableManager extends React.Component {
         description: PropTypes.string,
         id: PropTypes.string
       })
-    ).isRequired,
+    ),
```

Comment:
As a general practice, if a prop isn’t required it should have a default value in `defaultProps`.